components/coreapi/types.go

```diff
@@ -113,13 +113,13 @@ type nodeStatus struct {
 	LastConfirmedBlockID string `json:"lastConfirmedBlockId"`
 	// The latest finalized slot.
 	FinalizedSlot iotago.SlotIndex `json:"latestFinalizedSlot"`
-	// The Accepted Tangle Time
+	// The PreAccepted Tangle Time
```

Comment:
Are those really PreAccepted/PreConfirmed or is it an artifact of bulk renaming? In the clock I see that there is no PreAccepted time tracked.